uv.gyp

```diff
@@ -112,6 +112,7 @@
               '-liphlpapi',
               '-lpsapi',
               '-lshell32',
+              '-luser32',
```

Comment:
You also need to add this [here](https://github.com/libuv/libuv/blob/v1.x/configure.ac#L62) for MinGW.